python/python_fastapi_server/routers/netcdf_to_covjson.py

```diff
@@ -198,7 +239,7 @@ def netcdf_to_covjson(netcdfdataset) -> Coverage:
     import requests
 
     SERVICE = "https://geoservices.knmi.nl/adagucserver?"
-    SERVICE= "http://localhost:8080/adagucserver?"
+    SERVICE = "http://localhost:8080/adagucserver?"
```

Comment:
Should this stay? Is this code touched?

Reply:
Removed this not always working testcode